credoai/utils/credo_api_utils.py

```diff
@@ -7,25 +7,23 @@
 from dotenv import dotenv_values
 from json_api_doc import deserialize, serialize
 
+CREDO_URL = "https://api.credo-qa.com"
```

Comment:
The default prod URL is api.credo.ai, e.g. https://api.credo.ai/api/v1/credoai/me/setting

Credo AI employees would override to https://api.credo-qa.com for dogfooding on QA env